bin/docker-compose

```diff
@@ -18,6 +18,39 @@ fi
 
 source "$TOOLKIT_ROOT/lib/shared-functions.sh"
 
+function check_sharelatex_env_vars() {
+  set +o pipefail
+  local rc_occurrences=$(grep -o SHARELATEX_ "$TOOLKIT_ROOT/config/overleaf.rc" | wc -l | sed 's/ //g')
+  set -o pipefail
```

Comment:
Since the `$(...)` runs in a subshell you can temporarily disable pipefail inside the subshell like this:`local rc_occurences=$(set +o pipefail && grep ...)`. Then you don't need to reenable it afterward.

Reply:
Net! I upgraded the commands in https://github.com/overleaf/toolkit/pull/217/commits/f1a50d986e93d0fb3224b00d255e4a1492b755a5